pkg/policy/criteria/users.go

```diff
@@ -13,10 +13,7 @@ var usersBody = ast.Body{
 		session := get_session(input.session.id)
 	`),
 	ast.MustParseExpr(`
-		user := get_user(session)
-	`),
-	ast.MustParseExpr(`
-		user_id := user.id
```

Comment:
`get_user(session).id` is equivalent to `session.user_id` for normal sessions too, correct? 

My understanding is yes, but want to be sure this still works as expected for normal sessions.

https://github.com/pomerium/pomerium/blob/1a95036b8ce1d8c16c0b7ead9dcc43ce57835c75/pkg/policy/rules/rules.go#L29-L39

https://github.com/pomerium/pomerium/blob/1a95036b8ce1d8c16c0b7ead9dcc43ce57835c75/pkg/policy/rules/rules.go#L6-L27

Reply:
They are not equivalent. `session.user_id` will return the user id saved in the session or service account. `get_user(session).id` returns the id of the associated user. But if the user doesn't exist it would return nothing.